privaterelay/settings.py

```diff
@@ -90,6 +90,8 @@
     'https://www.google-analytics.com/',
     'https://accounts.firefox.com',
     'https://location.services.mozilla.com',
+    'https://basket-dev.allizom.org',
+    'https://basket.mozilla.org',
```

Comment:
issue (blocking): Only add `basket-dev.allizom.org` to dev environments. Something like:
```python
BASKET_SRC = 'https://basket-dev.allizom.org' if DEBUG or ON_HEROKU else 'https://basket.mozilla.org'
```
?

That means our stage environment would use production `basket.mozilla.org` though. How will basket handle that? Will it add a bunch of stage/QA users to the production lists? (Probably fine.)

Reply:
Oh, duh - no that's perfect. Add `BASKET_ORIGIN` is exactly what we want to enable per-environment CSP connections.

Reply:
Applied in 8c834a9d.